integration/__tests__/app.tests.ts

```diff
@@ -93,7 +96,10 @@ describe("Lens integration tests", () => {
       });
 
       it("ensures helm repos", async () => {
-        await app.client.waitUntilTextExists("div.repos #message-bitnami", "bitnami"); // wait for the helm-cli to fetch the bitnami repo
+        const { stdout: reposJson } = await promiseExec("helm repo list -o json");
```

Comment:
better than `spawnSync()`, which is already used elsewhere in this file?

Reply:
We probably should be using `exec` more. See this:

https://stackoverflow.com/questions/48698234/node-js-spawn-vs-execute